### DISCUSSION TITLE
How to edit the Welcome / Login page?

### DISCUSSION THREAD

**Discussion by tiagopfraga** [misskey-dev/misskey#14512] [Q&A]
Hello its possible to edit/change the Welcome guest / login page? if yes?
Where is locate in the script
tkns

**Comment by AmaseCocoa** [comment]
The root page for non-logged-in users seems to be located at [packages/frontend/src/components/MkVisitorDashboard.vue](https://github.com/misskey-dev/misskey/blob/develop/packages/frontend/src/components/MkVisitorDashboard.vue).
